extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition-sourceinfo",
```

Comment:
Please change it to
"$id": "https://ns.adobe.com/xdm/mixins/segmentdefinition-sourceinfo"

Reply:
Actually, this is not a standard XDM mixin so it should not be in the mixin namespace. The $id should be something like "https://ns.adobe.com/adobe/experience/audience-sharing/segmentdefinition-sourceinfo"

Reply:
Nevermind, what you have here is correct